pr.test.js

```diff
@@ -3,6 +3,8 @@ const { extractPRNumber, getReleaseType, getReleaseNotes, fetchPR } = require('.
 test('can extract a PR number from a PR merge commit message', () => {
     expect(extractPRNumber('Merge pull request #4 from some/mockBranch')).toEqual('4')
     expect(extractPRNumber('Merge pull request #42 from some/mockBranch')).toEqual('42')
+    expect(extractPRNumber('My PR squashed (#56)')).toEqual('56')
+    expect(extractPRNumber('Squash pr (#45)\n\n multiple commit messages appended')).toEqual('45')
```

Comment:
Since this unit test is specific to merge commits, let's move these two cases into their own new test for squash merges.

Reply:
I've split the pr message testcase out for the squash merge case.